src/examples/forms/multithumbrange/MultiThumbRangeValidation.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useCallback, useMemo } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Message, MultiThumbRange } from '@zendeskgarden/react-forms';
+
+type validationTypes = 'success' | 'warning' | 'error';
+
+const Example = () => {
+  const [minVal, setMinVal] = useState();
+  const [maxVal, setMaxVal] = useState();
+  const [validation, setValidation] = useState<validationTypes>('success');
+
+  const validationMessage = useMemo(() => {
+    if (validation === 'success') {
+      return 'Growing all the time';
+    } else if (validation === 'warning') {
+      return 'Growing regularly';
+    }
+
+    return 'Growing slowly';
+  }, [validation]);
+
+  const onChange = useCallback(({ minValue, maxValue }) => {
+    setMinVal(minValue);
+    setMaxVal(maxValue);
+
+    if (minValue <= 20) {
+      setValidation('success');
+    } else if (minValue > 20 && maxValue > 70) {
+      setValidation('warning');
+    } else {
+      setValidation('error');
+    }
+  }, []);
```

Comment:
You thinking we should update the content to correlate with the spacing? Or should we update the behavior to be a little simpler?

Reply:
I updated the example to change message type a little easier for now.

Reply:
I was thinking something more like `maxValue - minValue`:

```js
  const onChange = useCallback(({ minValue, maxValue }) => {
    setMinVal(minValue);
    setMaxVal(maxValue);

    const result = maxValue - minValue;

    if (result >= 70) {
      setValidation('success');
    } else if (result >= 30) {
      setValidation('warning');
    } else {
      setValidation('error');
    }
  }, []);
```

Otherwise, I can't really make sense of what the two controls are doing without looking at the code. With :point_up: it starts to feel a little more intuitive.

Reply:
Nice, ty